src/scss/components/_card.scss

```diff
@@ -40,7 +40,6 @@
 
   &--action {
     cursor: pointer;
-    padding-bottom: calc(44px + 16px);
```

Comment:
@eduardojmatos Actually, this is a revert. It is affecting already current card with an extra bottom space  👍